lib/moesif_rack/app_config.rb

```diff
@@ -55,14 +57,25 @@ def parse_configuration(config_api_response)
         end
     end
 
-    def get_sampling_percentage(config_api_response, user_id, company_id)
+    def get_sampling_percentage(event_model, config_api_response, user_id, company_id)
         # Get sampling percentage
         begin
             # Check if response body is not nil
             if !config_api_response.nil? then 
                 @helpers.log_debug("Getting sample rate for user #{user_id} company #{company_id}")
                 @helpers.log_debug(config_api_response.to_s)
 
+                # Get Regex Sampling rate
+                regex_config = config_api_response.fetch('regex_config', nil)
+
+                if !regex_config.nil? and !event_model.nil?
```

Comment:
Looks correct, regex should be higher priority than user/company level sampling.